staging/src/k8s.io/client-go/tools/cache/controller.go

```diff
@@ -147,6 +148,9 @@ func (c *controller) Run(stopCh <-chan struct{}) {
 	if c.config.WatchErrorHandler != nil {
 		r.watchErrorHandler = c.config.WatchErrorHandler
 	}
+	if s := os.Getenv("ENABLE_CLIENT_GO_WATCH_LIST_ALPHA"); len(s) > 0 {
```

Comment:
@deads2k do you have a preference on how to allow clients to enable/disable the streaming feature?

I'd like to avoid updating code generation at least for Alpha.